src/components/AuthenticationModal/agent.ts

```diff
@@ -0,0 +1,17 @@
+import { createAgent } from '@veramo/core'
+import { IQRCodeGenerator, QrCodeProvider } from '@sphereon/ssi-sdk-qr-react'
+import {
+  ISIOPv2OID4VPRPRestClient,
+  SIOPv2OID4VPRPRestClient
+} from '@sphereon/ssi-sdk.siopv2-oid4vp-rp-rest-client'
+
+const agent = createAgent<IQRCodeGenerator & ISIOPv2OID4VPRPRestClient>({
+  plugins: [
+    new QrCodeProvider(),
+    new SIOPv2OID4VPRPRestClient({
+      baseUrl: process.env.NEXT_PUBLIC_BACKEND_BASE_URL,
+      definitionId: process.env.NEXT_PUBLIC_PRESENTATION_DEF_ID
```

Comment:
Make it clear in the ENV vars that we are talking about SSI, to not have it confused with other ENV vars

Reply:
Fixed!